File: Instagram.java
import java.util.*;
class Codechef
{
    public static void main(String main[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-->0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            if(x>y*10)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}

File: Instagram.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y;
        scanf("%d%d",&x,&y);
        if(x>y*10)
        printf("YES\n");
        else
        printf("NO\n");
    }
}
